dist/generated/rum.d.ts

```diff
@@ -46,7 +46,7 @@ export declare type RumActionEvent = CommonProperties & {
             /**
              * Action frustration types
              */
-            readonly type: ('rage_click' | 'dead_click' | 'error_click')[];
+            readonly type: ('rage_click' | 'dead_click' | 'error_click' | 'rage_tap' | 'error_tap')[];
```

Comment:
**Question:** Shouldn't we do the same for frustration records, since it's not shared ?

Reply:
I only re-generated the definitions out of changes that were done previously in the schemas.

Maybe, in the future, but no need to do it now. It'll take some time until we see mobile recorders sending frustration records in replay payloads.

What you should ensure though, is for the player Frustration-handling logic to gracefully handle unknown frustration types.